examples/tenant_tutorial/tenant_tutorial/urls_public.py

```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import patterns, url
```

Comment:
This works since Django 1.8, I'm not sure if it works on Django 1.7, however, Django 1.7 is no longer supported - I don't think anybody who's interested in this package and needs a tutorial would develop in 1.7 or older...